public/components/GeoPlot.vue

```diff
@@ -1277,12 +1330,18 @@ export default Vue.extend({
   display: inline;
   position: absolute;
 }
+.map-toggle:hover {
```

Comment:
those `:hover` and `.active` can be factorized. Maybe by creating a `.map-control` class to use on all buttons.